src/renderer/renderElement.ts

```diff
@@ -793,7 +779,7 @@ const drawElementFromCanvas = (
         zoom,
     );
 
-    context.translate(cx * scaleXFactor, cy * scaleYFactor);
+    context.translate(cx, cy);
```

Comment:
scale doesn't apply to arrows currently, and if it ever does we'll need to rewrite half the code anyway